civil.go

```diff
@@ -146,7 +154,7 @@ func (d *Date) UnmarshalJSON(data []byte) error {
 	}
 	val, err := ParseDate(s)
 	if err != nil {
-		return fmt.Errorf("invalid date: %v", err)
+		return fmt.Errorf("invalid date, data: %s, err: %v", s, err)
```

Comment:
If we're already using pkg/errors, should we do `errors.Wrapf()` instead?

Reply:
yeah, that would be better.  Let me address that after I see what Chris has to say.

Go 1.13.x introduced a different way to do wrapped errors, which is now the "new standard", so probably should move away from `Wrapf` at some point.

Reply:
agreed w/Matt

Reply:
The new "standard" is to do use a fmt verb which IMO is extremely weird. We've got our own errors package that I think makes more sense, but can't use that here. `pkg/errors.Wrapf` is fine for this lib.